test/dotnet-test3.Tests/Test3Tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.Test3.Tests
+{
+    public class Test3Tests : TestBase
```

Comment:
We use a BDD named convention for our tests. Please see tests under dotnet-build3.Tests for instance for an example of that.